app/pages/home-for-user/news-section.jsx

```diff
@@ -0,0 +1,127 @@
+import React from 'react';
+import apiClient from 'panoptes-client/lib/api-client';
+import { Link } from 'react-router';
+import ProjectCard from '../../partials/project-card';
+import Publications from '../about/publications-page';
+
+import style from './news-section.styl';
+void style;
+
+const NewsSection = React.createClass({
+
+  getDefaultProps() {
+    return {
+      newDatasets: [],
+    };
+  },
+
+  componentDidMount() {
+    this.fetchProjects();
+    this.recentPublications();
+  },
+
+  getInitialState() {
+    return {
+      avatars: {},
+      projects: [],
+      publications: [],
+    };
+  },
+
+  recentPublications() {
+    const articles = []
+    for (var category in Publications.articles()) {
```

Comment:
I believe what you want to do here is iterate over the keys of the object, so I think a better way to write this would be (and once the data is pulled out into its own file):
```
Publications.keys().forEach((category) => { // do stuff } );
```